lib/autoload/active_record/associations/preloader/manual_has_many_through.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveRecord::Associations::Preloader
+  class ManualHasManyThrough < HasMany
```

Comment:
Anyway I'm going to get calculated_attributes merged in, so that should remove the need for manual preloading.